test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -90,29 +98,30 @@ function pushImgToNodes() {
 
     echo "Copying $IMG_NAME image to every node..."
     # Copy image to control-plane node
-    scp -F ssh-config $SAVED_IMG k8s-node-control-plane:/tmp/antrea-ubuntu.tar &
+    scp -F ssh-config $SAVED_IMG k8s-node-control-plane:/tmp/image.tar &
```

Comment:
Thanks for addressing this bug in the script. I will remove [the fix](https://github.com/vmware-tanzu/antrea/pull/1714/files#diff-dfff29dc14aef4916b5174cca2f75d929f54db26ed334bc48d7c50060a39046aR93) from my PR.